scripts/west_commands/tests/test_pyocd.py

```diff
@@ -16,20 +16,20 @@
 # parameters, to verify they're respected.
 #
 
-TEST_TOOL = 'test-tool'
+TEST_FLASH = 'test-flash'
 TEST_ADDR = 0xadd
 TEST_BOARD_ID = 'test-board-id'
 TEST_FREQUENCY = 'test-frequency'
 TEST_DAPARG = 'test-daparg'
 TEST_TARGET = 'test-target'
-TEST_FLASHTOOL_OPTS = ['--test-flashtool', 'args']
+TEST_FLASH_OPTS = ['--test-flash', 'args']
 TEST_SERVER = 'test-pyocd-gdbserver'
```

Comment:
I think this will need changes. This is being passed as the value for the `--gdbserver` option below down in TEST_ALL_KWARGS. But that option has been removed. I think instead if you add the `--pyocd` argument suggested above then this would actually be the value that you would pass for it as configured in that dict.

Reply:
Thanks. I didn't really understand how the test worked and wasn't able to run it locally (that's now been remedied), so I was flying blind. Pushed a new version, please have another look.